### DISCUSSION TITLE
Welcome to dotfiles Discussions!

### DISCUSSION THREAD

**Discussion by eleijonmarck** [eleijonmarck/dotfiles#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by a7ul** [comment]
Hello hello 👋🏽

**Reply by a7ul** [reply]
Pretty cool 😄

**Reply by eleijonmarck** [reply]
hey !
It can be made a thread

**Comment by eleijonmarck** [comment]
yoyo another comment
